client/controllers/deactivateUsersCtrl.js

```diff
@@ -4,8 +4,22 @@ module.exports = [
   '$scope',
   '$route',
   'userService',
+  'registerService',
   'alertService',
-  function ($scope, $route, userService, alertService) {
+  function ($scope, $route, userService, registerService, alertService) {
+    $scope.registers = [];
```

Comment:
Can also add a count above the table, so one can see how many have registered. This number should match the number of registrations on Abakus.